WcaOnRails/app/models/user_preferred_event.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
 class UserPreferredEvent < ActiveRecord::Base
   belongs_to :user
+  belongs_to :event
 
-  validates :event_id, inclusion: { in: Event.all_official.map(&:id) }
+  validates :event_id, inclusion: { in: Event.official.map(&:id) }
```

Comment:
@jonatanklosko Do I even need this validation now that we have the `belongs_to :event`?